### PR TITLE
Modify the wrong knowledge points

### PR DESCRIPTION
**Pull request by jimcat8**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by alemorvan** [comment]
Hello @jimcat8 
Pay attention not removing good information.

You are removing this sentence which is good:
In the GNU/Linux operating system, "block" is the smallest unit of storage in the file system,

It's okay to give the default size, but have you test it on differents version (8 & 9) and different fs ?

**Comment by alemorvan** [comment]
I mean, the block size is determined at the format stage I think.

**Comment by jimcat8** [comment]
> I mean, the block size is determined at the format stage I think.

Yep, you are right

File: docs/books/admin_guide/03-commands.md [hunk 1]
```diff
@@ -491,7 +491,7 @@ The `ls` command, however, has a lot of options (see `man`):
...
 | `-g`                                                         | Like -l option, but do not list owner.                                                                                               |
 | `-h`                                                         | Displays file sizes in the most appropriate format (byte, kilobyte, megabyte, gigabyte, ...). `h` stands for Human Readable. Needs to be used with -l option.         |
-| `-s`                                                         | Displays the allocated size of each file, in blocks. In the GNU/Linux operating system, "block" is the smallest unit of storage in the file system, one block equals 4096Byte. |
+| `-s`                                                         | Displays the allocated size of each file, in blocks. In the `ls` command, the default size of a single block is 1024Byte. In the GNU/Linux operating system, "block" is the smallest unit of storage in the file system, and generally speaking, one block is equal to 4096Byte.  |
```
Comment (SergeCroise) [inline docs/books/admin_guide/03-commands.md:494]:
```suggestion
| `-s`                                                         | Displays the allocated size of each file, in blocks. In the `ls` command, the default size of a single block is 1024 bytes. In the GNU/Linux operating system, "block" is the smallest unit of storage in the file system, and generally speaking, one block is equal to 4096 bytes.  |
```

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**